README.md

```diff
@@ -5,12 +5,13 @@ See [below](#collaborative-and-publication-workflows) for special cases like pub
 ## Software Prerequisites
 + `python3` v3.5.3+
 + `bedtools` (https://bedtools.readthedocs.io/en/latest/content/installation.html)
++ `Try::Tiny` Perl module
 + `IGOR` https://github.com/d3b-center/d3b-cli-igor
```

Comment:
Ha! IGOR has been deprecated in favor of https://github.com/Versent/saml2aws with directions to use here: https://www.notion.so/d3b/Setup-SAML-Login-1056131f1200806ba182f7b7c1793a40?pvs=4